corehq/apps/callcenter/sync_user_case.py

```diff
@@ -171,12 +175,22 @@ def _not_same(val1, val2):
     return changed_fields
 
 
+def get_sync_lock_key(user_id):
+    return ["sync_user_case_for_%s" % user_id]
+
+
 def sync_call_center_user_case(user):
+    helper = get_call_center_case_helper(user)
+    if helper:
+        helper.commit()
+
+
+def get_call_center_case_helper(user):
     config = user.project.call_center_config
     if config.enabled and config.config_is_valid():
```

Comment:
There's a smell in having this function implicitly return `None` when this condition evaluates to false. In the process of trying to think of a better solution I ended up doing some refactoring (https://github.com/dimagi/commcare-hq/pull/26118). I think the comments on that PR demonstrate why this one is not safe to merge.